data/hai/hai reveal 4 middle.txt

```diff
@@ -752,8 +753,8 @@ mission "Hai Reveal [A05] Take Teeneep to Hai-home"
 			
 			label fury
 			`	"WHAT!?"`
-			`	Teeneep's exclamation is loud enough that every head in the room turns towards you both. "The Elders would prioritize our secrecy over our safety, even when the purpose of the secrecy is to maintain our safety?! Some of them must have grown mold in their brains!"`
-			`	Teeneep seems to lapse into either some very colorful phrasing, or perhaps a string of Hai expletives. It's all in Hai, and completely unintelligible to you, but around the room you can hear much muttering and several Hai give each other pointed glances. Realizing this Teeneep gives them a look, composes herself and lowers her voice back to a tone of restrained fury.`
+			`	Teeneep's exclamation is loud enough that every head in the room turns towards you both. "The Elders would prioritize our secrecy over our safety, even when the purpose of the secrecy is to maintain our safety?!"`
+			`	It takes a few seconds for Teeneep to calm her fury before she continues.`
```

Comment:
Her childish behavior in this and A09 is inconsistent with her personality and achievements elsewhere in the campaign. I'm reducing her temper problems to "barely in control of her emotions" rather than "acts like a five-year-old" since it is more plausible.

Reply:
In the context of the other discussion about the deals this can be closed off

Reply:
I think you're asking me to delete this line (and the dialog does make sense without it) so I just did that.